Dockerfile

```diff
@@ -9,6 +9,20 @@
 
 FROM --platform=linux/amd64 python:3.12-slim-bookworm
 
+# OCI Labels 
+LABEL org.opencontainers.image.title="ScanCode Toolkit" 
+LABEL org.opencontainers.image.description="ScanCode Toolkit for code scanning and analysis." 
+LABEL org.opencontainers.image.url="https://github.com/nexB/scancode-toolkit" 
```

Comment:
We've now moved to repos over at https://github.com/aboutcode-org/scancode-toolkit, could you also replace these in all the labels?

Reply:
Also this can be the source URL fine, but not the image URL for sure. For example https://github.com/aboutcode-org/scancode-toolkit/pull/3027 could host this at ghcr.io and we could add a image URL from there then, we can remove `image.url` for now.